x-pack/plugins/observability_solution/entity_manager/docs/entity_definitions.md

```diff
@@ -16,9 +16,190 @@ To enable the backfill transform set a value to `history.settings.backfillSyncDe
 
 History and summary transforms will output their data to indices where history writes to time-based (monthly) indices (`.entities.v1.history.<definition-id>.<yyyy-MM-dd>`) and summary writes to a unique indice (`.entities.v1.latest.<definition-id>`). For convenience we create type-based aliases on top on these indices, where the type is extracted from the `entityDefinition.type` property. For a definition of `type: service`, the data can be read through the `entities-service-history` and `entities-service-latest` aliases.
 
-**Entity definition example**:
+#### Iterating on a definition
```

Comment:
There's overlap with the `Entity definition example` section below, can we somehow merge those two ?

Reply:
I moved a sentence or two around to make the reading transition smoother between the two sections but I think it's okay to have the first example contain those in line comments that repeat some of the information inline, WDYT?